docs/tutorial.md

```diff
@@ -456,21 +471,29 @@ except Exception as e:
     print (e)
 ```
 
-Therefore, we will need to refer to the *node* IDs attached to our sample
-to ensure that we extract only sample individuals.
+This is just telling us that some of the individuals we're trying
+to write to the VCF have nodes that are not samples.
+The reference to "missing" is a red herring:
+see {ref}`the tskit documentation <tskit:sec_data_model_missing_data>`
+for what it's talking about.
+So, instead of writing out genotypes of everyone alive,
+we need to get the list of alive individuals *whose nodes are samples*,
+using {meth}`is_sample() <tskit.Node.is_sample>`:
 
 ```{code-cell}
 indivlist = []
 for i in ts.individuals_alive_at(0):
     ind = ts.individual(i)
-    if ts.node(ind.nodes[0]).is_sample():                                                                              
+    if ts.node(ind.nodes[0]).is_sample():
        indivlist.append(i)
+       # if one node is a sample, the other should be also:
+       assert ts.node(ind.nodes[1]).is_sample()
 with open("example_snps.vcf", "w") as vcffile:
     ts.write_vcf(vcffile, individuals=indivlist)
 ```
 
 :::{note}
```

Comment:
I don't think we actually need this note any more; what do you think?

Reply:
Agreed, I think you covered it in the preceding paragraph